templates/github/ReadmeTemplate.md

```diff
@@ -0,0 +1,74 @@
+# Project name
+
+Here you can add a project description. Some questions that you can ask to make a description are:
+- What is this application?
+- What technology is used?
+- Is this project dependent on another service/project?
+
+## Table of Contents
+1. [Project documentation](#project-documentation)
+2. [Project setup](#project-setup)
+3. [Available scripts](#available-scripts)
+4. [Environments](#environments)
+
+## Project documentation
+- [Jira board](https://pondevelopment.atlassian.net/jira/projects)
```

Comment:
As this is a public repo, we shouldn't include this information in the README as it can be used for social engineering.